app/controllers/api/v1/emotions_controller.rb

```diff
@@ -40,15 +40,17 @@ def additional_params
     #  below in the response steps must be wrote with only such format in other case will be mistakes
     {
       current_user_id: current_user.id,
-      time_period: TimePeriod.current,
+      time_period: TimePeriod.find_or_create_time_period,
```

Comment:
@SerhiiBorozenets this is second call to db where you find the same value of TimePeriod, I think it's better to move it in separate method to reduce the number of db requests